<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - Default test</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>Default test</h1>
<p>
  Test duration: 73,619s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">test.java.TestTest</td>
      </tr>
            
<tr>
  <td class="method">
                  testHbogoWebdriverTest1
      </td>
  <td class="duration">
    50,735s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <br><p>Verification is not successfull<br></p>
                                Element is present
                                Element is present
                                Element is present
                                <a href="/Users/georgegaspar/Documents/Tests/Firefox-2015-1-6-0-59-26-725.png"><p align="left"Element screenshot at Fri Feb 06 00:59:32 CET 2015</p>
                                <p><img width="50%" src="/Users/georgegaspar/Documents/Tests/Firefox-2015-1-6-0-59-26-725.png" alt="screenshot at Fri Feb 06 00:59:32 CET 2015"/></p></a><br />
                                Element is present
                                Element is present
                                Element is present
                                Element is present
                                Completed executing following method : TestTest.testHbogoWebdriverTest1<br>
                                Completed executing following method : TestTest.testHbogoWebdriverTest1<br>
                                Test Method resides in test.java.TestTest<br>
                                <p>Test Status: </p><p>Failed</p>
                                <a href="/Users/georgegaspar/Documents/Tests/Firefox-testHbogoWebdriverTest1(test.java.TestTest)-2015-1-6-0-59-43-611.png"><p align="left">Error screenshot at Fri Feb 06 00:59:52 CET 2015</p>
                                <p><img width="50%" src="/Users/georgegaspar/Documents/Tests/Firefox-testHbogoWebdriverTest1(test.java.TestTest)-2015-1-6-0-59-43-611.png" alt="screenshot at Fri Feb 06 00:59:52 CET 2015"/></p></a><br />
                  </div>
    

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.Throwable: Multiple failures (3)<br/>
(1)java.lang.AssertionError:expected [true] but found [false]<br/>
(2)java.lang.AssertionError:expected [HB. Brhol. Bármikor] but found [HBO GO. Bárhol. Bármikor.]<br/>
(3)java.lang.NullPointerException:null<br/>
<br/>
Failure 1 of 3<br/>
java.lang.AssertionError: expected [true] but found [false]<br/>
	at org.testng.Assert.fail(Assert.java:94)<br/>
	at org.testng.Assert.failNotEquals(Assert.java:494)<br/>
	at org.testng.Assert.assertTrue(Assert.java:42)<br/>
	at org.testng.Assert.assertTrue(Assert.java:52)<br/>
	at test.java.TestBase.assertTrue(TestBase.java:41)<br/>
	at test.java.TestBase.verifySuccess(TestBase.java:83)<br/>
	at test.java.TestTest.testHbogoWebdriverTest1(TestTest.java:86)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/>
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/>
	at java.lang.reflect.Method.invoke(Method.java:483)<br/>
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)<br/>
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)<br/>
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)<br/>
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)<br/>
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br/>
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br/>
	at org.testng.TestRunner.privateRun(TestRunner.java:767)<br/>
	at org.testng.TestRunner.run(TestRunner.java:617)<br/>
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br/>
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)<br/>
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)<br/>
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)<br/>
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br/>
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)<br/>
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)<br/>
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)<br/>
	at org.testng.TestNG.run(TestNG.java:1057)<br/>
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)<br/>
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)<br/>
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)<br/>
<br/>
Failure 2 of 3<br/>
java.lang.AssertionError: expected [HB. Brhol. Bármikor] but found [HBO GO. Bárhol. Bármikor.]<br/>
	at org.testng.Assert.fail(Assert.java:94)<br/>
	at org.testng.Assert.failNotEquals(Assert.java:494)<br/>
	at org.testng.Assert.assertEquals(Assert.java:123)<br/>
	at org.testng.Assert.assertEquals(Assert.java:165)<br/>
	at test.java.TestBase.assertEquals(TestBase.java:61)<br/>
	at test.java.TestBase.verifyEquals(TestBase.java:141)<br/>
	at test.java.TestTest.testHbogoWebdriverTest1(TestTest.java:88)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/>
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/>
	at java.lang.reflect.Method.invoke(Method.java:483)<br/>
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)<br/>
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)<br/>
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)<br/>
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)<br/>
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br/>
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br/>
	at org.testng.TestRunner.privateRun(TestRunner.java:767)<br/>
	at org.testng.TestRunner.run(TestRunner.java:617)<br/>
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br/>
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)<br/>
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)<br/>
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)<br/>
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br/>
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)<br/>
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)<br/>
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)<br/>
	at org.testng.TestNG.run(TestNG.java:1057)<br/>
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)<br/>
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)<br/>
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)<br/>
<br/>
Failure 3 of 3<br/>
java.lang.NullPointerException</b></a><br />
    <div class="stackTrace" id="exception-0">
              test.java.TestTest.testHbogoWebdriverTest1(TestTest.java:169)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:483)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:714)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:767)<br />
              org.testng.TestRunner.run(TestRunner.java:617)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:334)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:240)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1149)<br />
              org.testng.TestNG.run(TestNG.java:1057)<br />
              org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)<br />
                </div>
      </td>
</tr>
      </table>


  
</body>
</html>
